src/js/nav-local.js

```diff
@@ -1,53 +1,49 @@
-/* eslint-disable no-unreachable */
+const widgetsToHideSelector = [
```

Comment:
I do not like that we now have this list in two places. Do you have any ideas how we could maintain this in only one place?

Reply:
Yeah so it turns out that this whole file is obsolete now that the nav-local has been converted to use the nav-toggle. Removed the file and at the same time this problem disappeared :)